templates/csharp/notification-http-timer-trigger-isolated/TeamsBot.cs.tpl

```diff
@@ -1,15 +1,31 @@
 using Microsoft.Bot.Builder;
 using Microsoft.Bot.Builder.Teams;
+using Microsoft.Bot.Schema;
 
 namespace {{SafeProjectName}}
 {
     /// <summary>
-    /// An empty bot handler.
+    /// Bot handler.
     /// You can add your customization code here to extend your bot logic if needed.
     /// </summary>
     public class TeamsBot : TeamsActivityHandler
     {
-        public override Task OnTurnAsync(ITurnContext turnContext, CancellationToken cancellationToken = default) =>
-            Task.CompletedTask;
+        public override async Task OnTurnAsync(ITurnContext turnContext, CancellationToken cancellationToken = default)
+        {
+            await base.OnTurnAsync(turnContext, cancellationToken);
+        }
+
+        protected override async Task OnMembersAddedAsync(IList<ChannelAccount> membersAdded, ITurnContext<IConversationUpdateActivity> turnContext, CancellationToken cancellationToken)
```

Comment:
Will this be called on each added member? For example:
- Adding bot to an existing group chat
- Adding User A to the group chat
- Adding User B to the group chat

Will bot send welcome to the group chat twice? Or send to A and B as 1:1 chat?

Reply:
The bot will send welcome message in these three cases. 
If developers want to customize this welcome message scope, developer can change the if condition for different scenarios based on user's need.
What we added in this template is just a general scenario.